boxoffice/views/admin_order.py

```diff
@@ -138,7 +138,8 @@ def order_api(org, order):
         'status': LINE_ITEM_STATUS[li.status],
         'base_amount': li.base_amount,
         'discounted_amount': li.discounted_amount,
-        'final_amount': li.final_amount
+        'final_amount': li.final_amount,
+        'assignee': format_assignee(li.current_assignee),
```

Comment:
Is this a text field? Doesn't it have additional data?

Reply:
`format_assignee` adds the assignee details(name, email, phone).

https://github.com/hasgeek/boxoffice/blob/66633b1f6766e831c7fa18365df6576d98c19dba/boxoffice/views/admin_order.py#L10